Makefile

```diff
@@ -37,8 +41,8 @@ dist-zip: clean
 	mkdir -p .$(DIST) # use a hidden directory so that it doesn't get copied into itself
 	cp -R * .$(DIST)
 	mv .$(DIST) $(DIST)
-	echo $(VERSION) > $(DIST)/.version # save the version so we can rerun the same tag
 	$(MAKE) -C $(DIST) install-vendor # install the composer vendor inside the dist dir
+	rm $(DIST)/Makefile # this step needs to be the last step before zipping
```

Comment:
Why does this need to be last? Shouldn't make a difference, as we never use the Makefile from the DIST?

Reply:
We still need the previous line to install the vendor packages. The zip is broken otherwise.

Reply:
Ah, yeah, of course 🤦‍♂️